### ISSUE TITLE
Certificate error when trying to run handler on windows

### ISSUE THREAD

**Issue by bigbam505** [DataDog/chef-handler-datadog#61]
So I was trying to run this on a windows host and I got the error show below.  I believe that this is a limitation of ruby on a windows platform.  I am not sure if there is a workaround for windows or plans to correct this but we would love to get chef runs into our datadog account on our windows hosts as well.

```
Running handlers:
[2015-04-30T18:09:59-04:00] INFO: Running report handlers SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
[2015-04-30T18:09:59-04:00] WARN: Could not submit event to Datadog (HTTP call failed): -1 SSL_connect returned=1 errno=0 state=SSLv3 read server certificate B: certificate verify failed
[2015-04-30T18:09:59-04:00] WARN: Could not submit ["env:ci"] tags for hostname.domain.net to Datadog: [-1, {}]
  - Chef::Handler::Datadog
Running handlers complete
[2015-04-30T18:09:59-04:00] INFO: Report handlers complete
```

**Comment by miketheman** [comment]
Thanks for reporting this. There's a number of Ruby-vs-Chef-vs-SSL issues - Windows is not a platform we've taken on yet to be fully supported by our Ruby api client library (which the handler depends on).

It's clearly a problem, and we hope to devote time towards developing a more robust client library, as well as improving our coverage of Windows.

**Comment by bigbam505** [comment]
So actually I did some more research this morning and was able to come up with a decent fix.  If you look at the chef client omnibus repo you will see something like [this](https://github.com/chef/omnibus-chef/blob/master/files/openssl-customization/windows/ssl_env_hack.rb).  Its included with every chef install along with the cacert.pem file that chef uses to fix the cert problem on windows.  What I did was set an environmental variable to `C:\opscode\chef\embedded\ssl\certs\cacert.pem` (the default install location for the chef cacert file on windows and everything worked correctly.  I just set this on all windows hosts with the following addition

```
   1
   2 # Setup the certs for ruby in windows
   3 env 'SSL_CERT_FILE' do
   4   action :create
   5   value "C:\\opscode\\chef\\embedded\\ssl\\certs\\cacert.pem"
   6 end
   7
```
